File: Examples/VisualBasic/WorkingWithTaskLinks/GetTaskLinkType.vb

Namespace WorkingWithTaskLinks
    Public Class GetTaskLinkType
        Public Shared Sub Run()
            ' ExStart:GetTaskLinkType
            ' Create project instance
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            Dim project1 As New Project(dataDir + "project.mpp")

            ' Display task link types 
            Dim allinks = project1.TaskLinks
            For Each tsklnk As TaskLink In allinks
                Console.WriteLine(tsklnk.LinkType.ToString())
            Next
            ' ExEnd:GetTaskLinkType
        End Sub
    End Class
End Namespace

File: Examples/CSharp/WorkingWithTaskLinks/GetTaskLinkType.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithTaskLinks
{
    using System;

    internal class GetTaskLinkType
    {
        public static void Run()
        {
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());
            
            //ExStart:GetTaskLinkType
            //ExFor: TaskLink.LinkType
            //ExSummary: Shows how to read task link types.
            var project = new Project(dataDir + "GetTaskLinkType.mpp");

            foreach (var link in project.TaskLinks)
            {
                Console.WriteLine(link.LinkType.ToString());
            }
            //ExEnd:GetTaskLinkType
        }
    }
}
